docs(release-skip): update guidelines to pnpm usage (#1255)
Frontify/fondue: CONTRIBUTING.md

## Running Fondue Locally

1. Clone the Git repository

> 2. Install the dependencies with

Reviewer: Maybe you could remove the `with` in that sentence or finish it. It's triggering my OCDs. 😅

Author: I forgot that `with` there. Sorry and thanks :D

3. Run Storybook
4. ???
5. Profit!

> > We use `pnpm`. If you don't have it installed in your local, please follow their installation guide https://pnpm.io/installation

Reviewer 2: ```suggestion
> Fondue uses `pnpm` to manage the dependencies. If you don't have it installed in your local, please refer to [their installation guide](https://pnpm.io/installation).
```

> ```

Reviewer 2: ```suggestion
```shell
```

$ git clone git@github.com:Frontify/fondue.git
$ pnpm i

> $ pnpm run storybook

Reviewer 2: ```suggestion
$ pnpm storybook
```

Theres a nifty script that creates that files you'll need:

> ```

Reviewer 2: ```suggestion
```shell
```
And same for the other code blocks

Author: Will leave these two blocks without the `shell` language set because they are not shell scripts:

```
<type>[optional scope]: <description>
```

And 

```
feat: Add CoolCatsAndKittens Component
```

> pnpm run component:create CoolCatsAndKittens

Reviewer 2: ```suggestion
pnpm component:create CoolCatsAndKittens
```
